e2e/node/package.json

```diff
@@ -5,13 +5,10 @@
   "type": "module",
   "scripts": {
     "ci": "npm run e2e",
-    "e2e": "vitest",
+    "e2e": "vitest run",
     "lint:fix": "npm run lint -- --fix",
     "lint": "eslint --ext '.js,.jsx,.ts,.tsx' basic canisters utils *.ts",
-    "mitm": "MITM=true vitest ./basic/mitm.test.ts",
-    "build": "",
```

Comment:
It's completely fair to take these out, but they were there to make it easier to fun `npm run build --ws` without errors getting thrown

Reply:
Ah I didn't realize that, thanks for explaining, I'll put them back.